props/saber_fett263_buttons.h

```diff
@@ -3216,11 +3222,15 @@ SaberFett263Buttons() : PropBase() {}
         }
         break;
       case MENU_VOLUME:
+#ifdef FETT263_CIRCULAR_VOLUME_MENU
+        ChangeVolume(direction);
```

Comment:
Consolidated to one function, these were user requests to be able to go from min to max or max to min by continuing dial turn in same direction instead of having to reverse all the way through the range.